server.js

```diff
@@ -0,0 +1,21 @@
+const http = require('http');
+const hr = require("./halls.js");
```

Comment:
odlično reusanje koda

Reply:
varijable je dobro nazivati po onome što predstavljaju. halls bi bilo preglednije nego hr